### ISSUE TITLE
<th> used in column context in Table fails to render last row bottom border on <td>

### ISSUE THREAD

**Issue by angrytongan** [vrimar/construct-ui#14]
When `<th>` is used in a column context, the final row bottom border does not display underneath `<td>` elements. When using `<th>` in a row context, or with `{ bordered: true }`,  it does render correctly.

```javascript
const {
    Table,

} = CUI;

const TableRendering = {
    view: () => {
        return m('', [
            m('', [
                m('p', m.trust('Table with <i>th</i> in top row')),
                m(Table, [
                    m('tr', [
                        m('th', 'heading row 1 column 1'),
                        m('th', 'heading row 1 column 2'),
                        m('th', 'heading row 1 column 3'),
                    ]),
                    m('tr', [
                        m('td', 'data row 2 column 1'),
                        m('td', 'data row 2 column 2'),
                        m('td', 'data row 2 column 3'),
                    ]),
                    m('tr', [
                        m('td', 'data row 3 column 1'),
                        m('td', 'data row 3 column 2'),
                        m('td', 'data row 3 column 3'),
                    ]),
                ]),
            ]),

            m('', [
                m('p', m.trust('Table with <i>th</i> in first column')),
                m(Table, [
                    m('tr', [
                        m('th', 'heading row 1 column 1'),
                        m('td', 'data row 1 column 2'),
                        m('td', 'data row 1 column 3'),
                    ]),
                    m('tr', [
                        m('th', 'heading row 2 column 1'),
                        m('td', 'data row 2 column 2'),
                        m('td', 'data row 2 column 3'),
                    ]),
                    m('tr', [
                        m('th', 'heading row 3 column 1'),
                        m('td', 'data row 3 column 2'),
                        m('td', 'data row 3 column 3'),
                    ]),
                ]),
            ]),

            m('', [
                m('p', m.trust('Table with <i>th</i> in second column')),
                m(Table, [
                    m('tr', [
                        m('td', 'data row 1 column 1'),
                        m('th', 'heading row 1 column 2'),
                        m('td', 'data row 1 column 3'),
                    ]),
                    m('tr', [
                        m('td', 'data row 2 column 1'),
                        m('th', 'heading row 2 column 2'),
                        m('td', 'data row 2 column 3'),
                    ]),
                    m('tr', [
                        m('td', 'data row 3 column 1'),
                        m('th', 'heading row 3 column 2'),
                        m('td', 'data row 3 column 3'),
                    ]),
                ]),
            ]),
        ]);
    },
};

document.addEventListener('DOMContentLoaded', () => {
    m.route(document.body, '/', {
        '/': TableRendering
    });
});
```

**Comment by vrimar** [comment]
Could you explain a bit further? This is what I'm seeing on latest Chrome Mac OS.

![Capture](https://user-images.githubusercontent.com/5660247/76808503-5a9e6a80-67be-11ea-8a9e-c5e4699a7384.PNG)

**Comment by angrytongan** [comment]
Latest Chrome on Mac OS X here. I'm running off the 1.0.0 branch.

![Screen Shot 2020-03-17 at 10 21 41 am](https://user-images.githubusercontent.com/1642050/76810384-9aeefe80-6839-11ea-8d5a-a76bff743a92.png)

**Comment by vrimar** [comment]
Fixed in v0.2.4.
